tomviz/python/tomviz/utils.py

```diff
@@ -154,6 +160,72 @@ def vtk_cast_map():
     return _vtk_cast_types
 
 
+def get_python_voxel_type(dataset):
+    """
+    Return the Python type that can represent the voxel type in the dataset.
+    The dataset can be either a VTK dataset or an ITK image.
+    """
+
+    # Try treating dataset as a VTK data set first.
+    try:
+        # Set up map between VTK data type and Python type
+        global _vtk_to_python_types
+
+        if _vtk_to_python_types == None:
+            import vtk
+            _vtk_to_python_types = {
+                vtk.VTK_UNSIGNED_CHAR: int,
+                vtk.VTK_CHAR: int,
+                vtk.VTK_UNSIGNED_SHORT: int,
+                vtk.VTK_SHORT: int,
+                vtk.VTK_UNSIGNED_INT: int,
+                vtk.VTK_INT: int,
+                vtk.VTK_UNSIGNED_LONG: long,
+                vtk.VTK_LONG: long,
+                vtk.VTK_FLOAT: float,
+                vtk.VTK_DOUBLE: float
+            }
+
+        pd = dataset.GetPointData()
+        scalars = pd.GetScalars()
+
+        return _vtk_to_python_types[scalars.GetDataType()]
+    except:
+        pass
```

Comment:
Can add the type of the exception ( KeyError? ) I , otherwise this might hide errors such  as being unable to import vtk?